rust/sdk-processor/src/parquet_processors/parquet_token_v2_processor.rs

```diff
@@ -0,0 +1,182 @@
+}
+
+impl ParquetTokenV2Processor {
+    pub async fn new(config: IndexerProcessorConfig) -> anyhow::Result<Self> {
+        let db_pool = initialize_database_pool(&config.db_config).await?;
+        Ok(Self { config, db_pool })
+    }
+}
+
+#[async_trait::async_trait]
+impl ProcessorTrait for ParquetTokenV2Processor {
+    fn name(&self) -> &'static str {
+        self.config.processor_config.name()
+    }
+
+    async fn run_processor(&self) -> anyhow::Result<()> {
+        // Run Migrations
+        let parquet_db_config = match self.config.db_config {
+            DbConfig::ParquetConfig(ref parquet_config) => {
+                run_migrations(
+                    parquet_config.connection_string.clone(),
+                    self.db_pool.clone(),
+                )
+                .await;
+                parquet_config
+            },
+            _ => {
+                return Err(anyhow::anyhow!(
+                    "Invalid db config for ParquetTokenV2Processor {:?}",
+                    self.config.db_config
+                ));
+            },
+        };
+
+        // Check and update the ledger chain id to ensure we're indexing the correct chain
+        let grpc_chain_id = TransactionStream::new(self.config.transaction_stream_config.clone())
+            .await?
+            .get_chain_id()
+            .await?;
+        check_or_update_chain_id(grpc_chain_id as i64, self.db_pool.clone()).await?;
+
+        let parquet_processor_config = match self.config.processor_config.clone() {
+            ProcessorConfig::ParquetTokenV2Processor(parquet_processor_config) => {
+                parquet_processor_config
+            },
+            _ => {
+                return Err(anyhow::anyhow!(
+                    "Invalid processor configuration for ParquetTokenV2Processor {:?}",
+                    self.config.processor_config
+                ));
+            },
+        };
+
+        let processor_status_table_names = self
+            .config
+            .processor_config
+            .get_processor_status_table_names()
+            .context("Failed to get table names for the processor status table")?;
+
+        let starting_version = get_min_last_success_version_parquet(
+            &self.config,
+            self.db_pool.clone(),
+            processor_status_table_names,
+        )
+        .await?;
+        println!("Starting version: {:?}", starting_version);
+
+        // Define processor transaction stream config
+        let transaction_stream = TransactionStreamStep::new(TransactionStreamConfig {
+            starting_version: Some(starting_version),
+            ..self.config.transaction_stream_config.clone()
+        })
+        .await?;
+
+        let backfill_table = set_backfill_table_flag(parquet_processor_config.backfill_table);
+        // TODO: Update this
+        let parquet_token_v2_extractor =
+            ParquetTokenV2Extractor::new(backfill_table, 5, 500, self.db_pool.clone());
```

Comment:
How come these values aren't taken from the config?

Reply:
those were just placeholders, and this code is updated in the stacked pr as well